subsys/lorawan/nvm/lorawan_nvm.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2022 Intellinium <giuliano.franchetto@intellinium.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef LORAWAN_NVM_H
```

Comment:
Other include files use guards in this format
`#ifndef ZEPHYR_INCLUDE_RANDOM_RAND32_H_`
so probably change this to
`ZEPHYR_INCLUDE_LORAWAN_NVM_H_`

Reply:
done